website/static/js/share/results.js

```diff
@@ -37,79 +38,155 @@ Results.controller = function(vm) {
     self.vm = vm;
     self.vm.resultsLoading = m.prop(false);
 
+    self.renderTitleBar = function(result) {
+        return [m(
+            'a[href=' + result.uris.canonicalUri + ']', ((result.title || '').length > 0) ? m.trust(result.title) : 'No title provided'),
```

Comment:
Hmm. I suppose I was operating under that assumption as well. I had the m.trust so that the elasticsearch highlighting would work. It's pretty minor, so I can just remove these and we can revisit highlighting at a later date.